sdk/documenttranslation/Azure.AI.DocumentTranslation/README.md

```diff
@@ -1,19 +1,488 @@
 # Azure Cognitive Services Document Translation client library for .NET
+Document Translation is a cloud-based feature of the Azure Translator service and is part of the Azure Cognitive Service family of REST APIs. The Document Translation API translates documents to and from 90 languages and dialects while preserving document structure and data format.
+
+[Source code][documenttranslation_client_src] | [Package (NuGet)][documenttranslation_nuget_package] | [API reference documentation][documenttranslation_refdocs] | [Product documentation][documenttranslation_docs] | [Samples][documenttranslation_samples]
 
 ## Getting started
 
 ### Install the package
+Install the Azure Document Translation client library for .NET with [NuGet][nuget]:
+
+```PowerShell
+dotnet add package Azure.AI.DocumentTranslation
+```
 
 ### Prerequisites
+* An [Azure subscription][azure_sub].
+* An existing Cognitive Services or Translator resource.
+
+#### Create a Cognitive Services or Translator resource
+Translator supports both [multi-service and single-service access][cognitive_resource_portal]. Create a Cognitive Services resource if you plan to access multiple cognitive services under a single endpoint/key. For Translator access only, create a Translator resource.
+
+You can create either resource using: 
+
+**Option 1:** [Azure Portal][cognitive_resource_portal].
+
+**Option 2:** [Azure CLI][cognitive_resource_cli]. 
+
+Below is an example of how you can create a Translator resource using the CLI:
+
+```PowerShell
+# Create a new resource group to hold the Translator resource -
+# if using an existing resource group, skip this step
+az group create --name <your-resource-name> --location <location>
+```
+
+```PowerShell
+# Create Translator
+az cognitiveservices account create \
+    --name <your-resource-name> \
+    --resource-group <your-resource-group-name> \
+    --kind TextTranslation \
+    --sku <sku> \
+    --location <location> \
+    --yes
+```
+For more information about creating the resource or how to get the location and sku information see [here][cognitive_resource_cli].
 
 ### Authenticate the client
+In order to interact with the Document Translation service, you'll need to create an instance of the [DocumentTranslationClient][documenttranslation_client_class] class. You will need an **endpoint**, and an **API key** to instantiate a client object.  For more information regarding authenticating with cognitive services, see [Authenticate requests to Azure Cognitive Services][cognitive_auth].
+
+#### Get API Key
+You can get the `endpoint` and `API key` from the Cognitive Services resource or Translator resource information in the [Azure Portal][azure_portal].
```

Comment:
because getting the endpoint for the service is currently not straight forward, consider having a separate section for endpoint and follow what Python is doing (with the new comments I put there)